HopsanGenerator/src/GeneratorTypes.cpp

```diff
@@ -728,3 +729,30 @@ QString BuildFlags::platformString() const
 //{
 //    return compilerString(mCompiler);
 //}
+
+void getParameters(QList<ParameterSpecification> &parameters, hopsan::ComponentSystem *pSystem)
+{
+    for(hopsan::ParameterEvaluator *par : (*pSystem->getParametersVectorPtr())) {
+        ParameterSpecification spec;
+        if(par->getType() == "double") {
+            spec.type = "Real";
+        }
+        else if(par->getType() == "integer") {
+            spec.type = "Integer";
+        }
+        else if(par->getType() == "bool") {
+            spec.type = "Boolean";
+        }
+        else if(par->getType() == "string") {
+            spec.type = "String";
+        }
+        else {
```

Comment:
Should we support the other types as well,

Textblock and Conditional (the combobox type), Textblock could perhaps be treated as string and conditional as int

Reply:
Ok, I added them, but they are not tested yet. We can add some more advanced test cases later.